platform-hub-api/db/structure.sql

```diff
@@ -265,7 +265,9 @@ CREATE TABLE kubernetes_clusters (
     s3_secret_access_key character varying NOT NULL,
     s3_object_key character varying NOT NULL,
     created_at timestamp without time zone NOT NULL,
-    updated_at timestamp without time zone NOT NULL
+    updated_at timestamp without time zone NOT NULL,
+    api_url character varying NOT NULL,
```

Comment:
inject two new fields before create_at pls.

Reply:
This is auto-generated

Reply:
:) yeah, missed that completely...